### ISSUE TITLE
"Xamarin.Forms" button on leftpane is not having color contrast ratio of 4.5:1.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#299]
### Pre-requisites:
Launch the Visual Studio for Mac and Sign-in with Enterprise account.

Create projects:

1. Go to Visual Studio-> File-> New Solution. -> Multiplatform-> App-> Xamarin.Forms-> Blank Forms App.
2. Hit on Next button to create the project.
3. Hit on Debug(Play)/Build button to build/deploy the solution/project.
4. As per the choice/requirements, Change the Device platform like iOS/ Android and iPhone X iOS 11.3 etc.
5. Hit on Inspector icon button (i.e., in the top toolbar buttons of the VS) to inspect the project.
6. On hitting on inspector button, Xamarin Inspector is Launched.

### Repro Steps:
1. Follow the pre-requsites to launch the Xamarin Inspector from visual studio.
2. Change the View to Inspector View.
3. Check the color contrast ratio of "Xamarin.Forms" of left pane.
### Expected Behavior
Color contrast ratio should be minimum 4.5:1
### Actual Behavior
Color contrast ratio is less than 4.5:1 i.e, 4:1
See the screenshots attached
### User Impact
Low vision user will feel dificult in seeing the text

![issue 3](https://user-images.githubusercontent.com/22956130/39991863-88a4ab3e-578e-11e8-95ab-587c1402dbb5.png)

### Environment
Visual Studio for Mac Enterprise [Version: 7.5 Preview (7.5 build 1222)]
Xamarin Inspector [Version: 1.4 (master/b3f92f9)]

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1830
